### ISSUE TITLE
Add method take 2 arguments

### ISSUE THREAD

**Issue by xbgmsharp** [christian-putzke/CardDAV-PHP#12]
I modified the add method to pass the UUID of the vCard. This it can be or not generated.

```
    public function add($vcard, $vcard_id = false)
    {
            if (!$vcard_id) {
                    $vcard_id = $this->generate_vcard_id();
            }
           $vcard = $this->clean_vcard($vcard);
          ....
 }
```

**Comment by xbgmsharp** [comment]
any update?

**Comment by christian-putzke** [comment]
Will be included in the next version 0.6. Should be released this weekend.
